docs/src/user-guide/format-struct-logs-syntax.md

```diff
@@ -24,10 +24,14 @@ Defines the key of the field whose value should replace the placeholder.
 
 * Nested fields can be specified using periods (`.`) to denote hierarchy.
   * E.g., the field `{"a:" {"b": 0}}` may be denoted by `a.b`.
+* Auto-generated fields in a
+[Key-Value Pair IR Stream](https://docs.yscope.com/clp/main/dev-guide/design-key-value-pair-ir-stream.html)
```

Comment:
Let's create a reference for this at the bottom of the page. Even though the link is not finalized, it will definitely be too long to include inline.